.circleci/config.yml

```diff
@@ -32,7 +32,14 @@ commands:
     steps:
       - attach_workspace:
           at: /tmp/workspace
-      - run: docker load -i /tmp/workspace/image.tar
+      - run:
+          name: Load agent image
+          command: |
+            if [ -f ~/.skip ]; then
```

Comment:
For conciseness sake, I would make this `[ -f ~/.skip ] && echo "Found ~/.skip, skipping docker image load!" >&2 && exit 0` so that it takes up less vertical space and makes the docker load command more prominent.  It is a common enough idiom in shell scripts that it shouldn't be too cryptic.